src/HotChocolate/AspNetCore/src/AspNetCore/GraphQLServerOptions.cs

```diff
@@ -23,6 +23,11 @@ public class GraphQLServerOptions
         /// </summary>
         public bool EnableGetRequests { get; set; } = true;
 
+        /// <summary>
+        /// Defines if GraphQL HTTP Multipart requests are allowed.
+        /// </summary>
+        public bool EnableMultipartRequests { get; set; } = true;
```

Comment:
Difficult since we do not know the schema when we build the middleware. This is transport which is configured and created before the schema is created.